zebrad/tests/common/lightwalletd/wallet_grpc.rs

```diff
@@ -41,8 +42,9 @@ pub fn spawn_lightwalletd_with_rpc_server(
         .with_failure_regex_iter(lightwalletd_failure_messages, lightwalletd_ignore_messages);
 
     lightwalletd.expect_stdout_line_matches("Starting gRPC server")?;
-    lightwalletd.expect_stdout_line_matches("Waiting for block")?;
-
+    if wait_for_blocks {
+        lightwalletd.expect_stdout_line_matches("Waiting for block")?;
```

Comment:
So i had to add this as this line is not happening on my test but also wanted  `send_transaction_test` to keep looking for it.

Reply:
I think it's fine, as long as the gRPC test results don't depend on being exactly at the chain tip.